job-dsls/jobs/osbs/operators_osbs_seed_cloud_jobs.groovy

```diff
@@ -0,0 +1,207 @@
+            stringParam('CEKIT_BUILD_OPTIONS', "${cekitBuildOptions}")
+            stringParam('KERBEROS_PRINCIPAL', "${kerberosPrincipal}")
+            stringParam('OSBS_BUILD_USER', "${osbsBuildUser}")
+            stringParam('KERBEROS_KEYTAB', "${kerberosKeytab}")
+            stringParam('KERBEROS_CRED', "${kerberosCred}")
+            stringParam('IMAGE_REPO', "${imageRepo}")
+            stringParam('IMAGE_BRANCH', "${imageBranch}")
+            stringParam('IMAGE_SUBDIR', "${imageSubdir}")
+            stringParam('GIT_USER', "${gitUser}")
+            stringParam('GIT_EMAIL', "${gitEmail}")
+            stringParam('CEKIT_CACHE_LOCAL', "${cekitCacheLocal}")
+            stringParam('VERBOSE', "${verbose}")
+            stringParam('GITHUB_ORG_UNIT', "${githubOrgUnit}")
+            stringParam('RHPAM_KOGITO_OPERTOR_BRANCH', "${rhpamKogitoOperatorBranch}")
+            stringParam('BAMOE_BA_OPERTOR_BRANCH', "${bamoeBAOperatorBranch}")
+            stringParam('BAMOE_KOGITO_OPERTOR_BRANCH', "${bamoeKogitoOperatorBranch}")
+        }
+
+        definition {
+            cps {
+
+                script('''
+
+                      library 'jenkins-pipeline-shared-libraries'
+                      
+                      private void validateParameters(required, optionals){
+                          // Check if all required params are supplied
+                          for ( param in required ) {
+                              def arg = param.value[0]
+                              def flag = param.value[1]
+                              if(!param.value[0]){
+                                  error "${param.key} parameter is required but was not specified."
+                              }
+                          }
+                      }
+                      
+                      // Function to retrieve from the PROD_COMPONENT name the related repo name into the kiegroup org
+                      private String getOperatorRepoName(prodComponent){
+                          switch(prodComponent){
+                            case { it.endsWith('ba-operator') }:
+                                return 'kie-cloud-operator'
+                            case { it.endsWith('kogito-operator') }:
+                                return 'kogito-operator'
+                            default:
+                               error "${prodComponent} not supported."
+                          }
+                      }
+                      
+                      // Function to retrieve from the PROD_COMPONENT name the related repo branch
+                      private String getOperatorBranch(prodComponent){
+                          switch(prodComponent){
+                            case { it.startsWith('rhpam-ba') }:
+                                return 'main'
+                            case { it.startsWith('rhpam-kogito') }:
+                                return RHPAM_KOGITO_OPERTOR_BRANCH
```

Comment:
shouldn't these values be rhpamKogitoOperatorBranch or Constants.RHPAM_KOGITO_OPERTOR_BRANCH?
I think the first option would be better, wdyt?

Reply:
I'm defining that variable using the [stringParam](https://jenkinsci.github.io/job-dsl-plugin/#method/javaposse.jobdsl.dsl.helpers.BuildParametersContext.stringParam) function at [line 58](https://github.com/kiegroup/kie-ci/pull/1407/files#diff-02c3e3fd16762232fc76b897b899c1f763fb313735a8159e243aced4da65eb44R58) using the same approach we used for the other variables. Should we change it?

Reply:
IMHO this is fine as it is, we are using the same approach on all other jobs